crates/turbopack-ecmascript-runtime/js/src/build/runtime.ts

```diff
@@ -91,6 +95,25 @@ var relativeURL = function(this: any, inputUrl: string) {
 
 relativeURL.prototype = URL.prototype;
 
+/**
+ * Returns an absolute path to the given module's id.
+ *
+ */
+function createResolvePathFromModule(resolver: (moduleId: string) => Exports): (moduleId: string) => string {
+  return function resolvePathFromModule(moduleId: string): string {
+    const exported = resolver(moduleId);
+    const exportedPath = exported?.default ?? exported;
+    if (typeof exportedPath !== 'string') {
+      return exported as any;
+    }
+
+    const strippedAssetPrefix = exportedPath.replace(ASSET_PREFIX, '');
```

Comment:
The `ASSET_PREFIX` starts at the beginning, right? Can this just be a `slice`?

Reply:
Yes, I think so.